### PR TITLE
Update generateMarkdown.ts

### PR DESCRIPTION
**Pull request by timthedev07**
#1977

### CODE REVIEW

File: packages/tsconfig-reference/scripts/cli/generateMarkdown.ts [hunk 1]
```diff
@@ -56,8 +56,8 @@ languages.forEach((lang) => {
     <thead>
     <tr>
       <th>Flag</th>
-      <th>Type</th>
-      ${opts?.noDefaults ? "" : "<th>Default</th>"}
+      <th>Type</th>${opts?.noDefaults ? "" : "     <th>Default</th>"}
+
```
Comment (orta) [inline packages/tsconfig-reference/scripts/cli/generateMarkdown.ts:60]: Would this newline cause the same problem?

Comment (timthedev07) [reply packages/tsconfig-reference/scripts/cli/generateMarkdown.ts:60]: No, it's fine.

**Comment by timthedev07** [comment]
Sorry, just added the missing new line character in.

**Comment by orta** [comment]
Thanks!

**Pull request merged**